napari/utils/transforms/_tests/test_transforms.py

```diff
@@ -366,3 +372,22 @@ def test_affine_rotate_3d():
         ),
         a.rotate,
     )
+
+
+@pytest.mark.parametrize('AffineType', affine_type)
+def test_empty_units(AffineType):
+    assert AffineType(ndim=2).units == (PIXEL, PIXEL)
+    assert AffineType(ndim=3).units == (PIXEL, PIXEL, PIXEL)
+    assert AffineType(ndim=2).voxel_size == (1 * PIXEL, 1 * PIXEL)
+    assert AffineType(ndim=3).voxel_size == (1 * PIXEL, 1 * PIXEL, 1 * PIXEL)
+
+
+@pytest.mark.parametrize('AffineType', affine_type)
+def test_set_units(AffineType):
+    assert AffineType(ndim=2, units=('cm', 'mm')).units == (REG.cm, REG.mm)
+
+
+@pytest.mark.parametrize('AffineType', affine_type)
+def test_empty_axis_labels(AffineType):
+    assert AffineType().axis_labels == ('axis 1', 'axis 0')
+    assert AffineType(ndim=3).axis_labels == ('axis 2', 'axis 1', 'axis 0')
```

Comment:
I think these should be [-3, -2, -1]? This would make them compatible with NumPy indexing.

Reply:
updated